iocage/cli/get.py

```diff
@@ -63,11 +63,13 @@ def cli(
     host = iocage.lib.Host.Host(logger=logger)
 
     if _pool is True:
+        exit_status = 0
         try:
             print(host.datasets.active_pool.name)
         except Exception:
+            exit_status = 1
```

Comment:
we can exit here directly with `exit(1)` and exit with 0 later on so that the exit_status variable is not required.

Reply:
fair